specification/monitor/resource-manager/Microsoft.Insights/stable/2016-06-01/metrics_API.json

```diff
@@ -0,0 +1,265 @@
+            "required": true
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Successful request to get the list of metric values",
+            "schema": {
+              "$ref": "#/definitions/MetricCollection"
+            },
+            "examples": {
+              "application/json": {
+                "value": [
+                  {
+                    "data": [
+                      {
+                        "timeStamp": "2016-11-23T19:14:00Z",
+                        "total": 1.0
+                      },
+                      {
+                        "timeStamp": "2016-11-23T19:15:00Z"
+                      },
+                      {
+                        "timeStamp": "2016-11-23T19:16:00Z"
+                      },
+                      {
+                        "timeStamp": "2016-11-23T19:17:00Z"
+                      },
+                      {
+                        "timeStamp": "2016-11-23T19:18:00Z"
+                      },
+                      {
+                        "timeStamp": "2016-11-23T19:19:00Z",
+                        "total": 1.0
+                      },
+                      {
+                        "timeStamp": "2016-11-23T19:20:00Z"
+                      },
+                      {
+                        "timeStamp": "2016-11-23T19:21:00Z"
+                      },
+                      {
+                        "timeStamp": "2016-11-23T19:22:00Z"
+                      },
+                      {
+                        "timeStamp": "2016-11-23T19:23:00Z"
+                      },
+                      {
+                        "timeStamp": "2016-11-23T19:24:00Z",
+                        "total": 1.0
+                      }
+                    ],
+                    "id": "/subscriptions/df602c9c-7aa0-407d-a6fb-eb20c8bd1192/resourceGroups/andy0802australia/providers/Microsoft.Logic/workflows/andy0803australia/providers/microsoft.insights/metrics/RunsStarted",
+                    "name": {
+                      "value": "RunsStarted",
+                      "localizedValue": "Runs Started"
+                    },
+                    "type": "Microsoft.Insights/metrics",
+                    "unit": "2"
+                  }
+                ]
+              }
+            }
```

Comment:
examples usually link to the example file, For example:
https://github.com/Azure/azure-rest-api-specs/blob/f1d8c4ebb54fdc1301eb2f2a465982cf58b54f69/specification/network/resource-manager/Microsoft.Network/stable/2020-11-01/customIpPrefix.json#L142-L147

Reply:
Okay I've removed the inline example in favor of the existing example file already in the examples directory